backend/src/main/java/com/bakdata/conquery/apiv1/frontend/FEFilterType.java

```diff
@@ -11,5 +11,7 @@
 	REAL_RANGE,
 	NONE,
 	STRING,
-	MONEY_RANGE
+	MONEY_RANGE,
+	POSTAL_CODE,
```

Comment:
Postal Code oder Compound?

Es muss eins von Beiden sein, und wie schon gesagt, der Code soll am Ende in Eva sein, wo du es nicht einfach schaffen wirst ein weiteres Feld ins Enum zu kriegen.